Library.Wasm/OpenAPIs/LibraryApiWrapper.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Library.Wasm.OpenAPIs;
+
+/// <summary>
+/// Обертка для доступа к API, предоставляющая CRUD-операции над основными сущностями и аналитические методы.
+/// </summary>
+public class LibraryApiWrapper(IConfiguration configuration)
+{
+    public readonly Client _client=new(configuration["Api:Url"], new HttpClient());
+
+    #region Books CRUD
+    public Task<ICollection<BookGetDto>> GetAllBooks() => _client.BookAllAsync();
```

Comment:
Во все методы добавить CancellationToken
```suggestion
    public Task<ICollection<BookGetDto>> GetAllBooks(CancellationToken ct = default) => _client.BookAllAsync(ct);
```